src/types/resource.types.ts

```diff
@@ -166,7 +166,7 @@ export enum ResourceFeatureNames {
 export enum ContributorFeatureNames {
   Type = 'dlr_contributor_type',
   Name = 'dlr_contributor_name',
-  Institution = 'institution',
+  Institution = 'HostingInstitution',
```

Comment:
heter featuren "Hostinginstitution" ?

Reply:
Copy-paste fra https://dlrdev.unit.no/

Reply:
men HostingInstitution er ikke et "featureName", så den bør helt ligge et annet sted

Reply:
Hvor finner jeg liste med featureName? Stakk av med contributorTypesene til dlrdev, så hvis det er en annen liste jeg skal bruke så trenger jeg den.

Reply:
de ligger vel gjemt i koden i backend et sted - jeg fikk nikolai til å fiske de fram for meg. Kanskje det er lettere nå som nesten alt ligger i monorepo

Reply:
Ser at man kan skrive inn hva som helst og det blir lagret på serveren (inkludert button mashing). Sent spørsmål til Stine og hun skal høre med referansegruppa om det var andre contributorTypes de vil ha.

Reply:
virker som om de er disse:     `private final static List<String> valid_features_project_contributor = asList("dlr_contributor_name", "dlr_contributor_type", "dlr_contributor_entity_type");`

Reply:
fant også : dlr_contributor_order og dlr_contributor_time_created

Reply:
Ahh sånn sett. Skal få fikset det. Tenkte du mente at 'HostingInstitution' var ikke ok, men 'institution' var det. Jeg er teit

Reply:
Skal få ordnet opp i det med en gang.

Reply:
nei nei nei - slik må du ikke si :-P

Reply:
Fikset!

Reply:
(men nå er det en hardkodet streng der istedet). Lurte på om jeg skulle gå i json fila å hente ut verdien men det virket så knødd.

`contributorTypeList['HostingInstitution']` eller `contributorTypeList[contributorTypeList.findIndexOf('HostingInstitution')` er ikke bedre enn 'HostingInstitution' hardkodet.